bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/Persistence/Propel/Mapper/EntityToTransferMapper.php

```diff
@@ -119,11 +112,28 @@ public function fromErpOrderAddressToTransfer(
         //ToDo: handle region
 
         return $erpOrderAddressTransfer
-            ->setCountry($this->countryFacade->getCountryByIdCountry($erpOrderAddress->getFkCountry()))
+            ->setCountry($this->fromCountryToTransfer($erpOrderAddress->getCountry()))
             ->setCreatedAt($this->convertDateTimeToTimestamp($erpOrderAddress->getCreatedAt()))
             ->setUpdatedAt($this->convertDateTimeToTimestamp($erpOrderAddress->getUpdatedAt()));
     }
 
+    /**
+     * @param \Orm\Zed\Country\Persistence\SpyCountry $countryEntity
+     * @param \Generated\Shared\Transfer\CountryTransfer|null $countryTransfer
+     *
+     * @return \Generated\Shared\Transfer\CountryTransfer
+     */
+    public function fromCountryToTransfer(
```

Comment:
Is that cool to have the entity outside the persistence layer?

Reply:
Good point. Thx @sourcecube